examples/plain/README.md

```diff
@@ -52,3 +52,45 @@ A robust example of how to make best use of the Bugsnag React Native notifier.
    ```
    react-native run-android --variant=release
    ```
+
+## Deobfuscating stacktraces
+
+This app provides working examples of how to deobfuscate stacktraces for JS, Android, and iOS. For further information, please consult [our React Native documentation](https://docs.bugsnag.com/platforms/react-native/showing-full-stacktraces/).
+
+### JavaScript
+
+You will need to upload source maps to Bugsnag in order to deobfuscate JavaScript stack traces. Example scripts for achieving this on Android/iOS in both debug and release builds can be found [here](scripts). The example script requires the installation of [bugsnag-sourcemaps](https://github.com/bugsnag/bugsnag-sourcemaps) to work. The scripts can also be invoked in this example project using [npm run](https://docs.npmjs.com/cli/run-script.html):
```

Comment:
Rather than say it requires the installation of `bugsnag-sourcemaps`, add it as a dev dependency to the project. See other comment.

Reply:
I've updated the wording here slightly to state that it's installed in this example project as a dev dependency.